tests/acceptance/dex/idtokenprovider.go

```diff
@@ -44,48 +50,55 @@ func (p *dexIdTokenProvider) implicitFlow() (map[string]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	if authorizeResp.StatusCode < 300 || authorizeResp.StatusCode > 399 {
-		return nil, fmt.Errorf("Authorize - response error: '%s' - %s", authorizeResp.Status, readRespBody(authorizeResp))
+	if authorizeResp.StatusCode < 200 || authorizeResp.StatusCode > 399 {
+		return nil, errors.New(fmt.Sprintf("Authorize - response error: '%s' - %s", authorizeResp.Status, readRespBody(authorizeResp)))
```

Comment:
why? in line 67 you left the `fmt.Errorf` and just make the first letter lowercase. Please unify your approach

Reply:
fixed